snuba/state/__init__.py

```diff
@@ -207,13 +208,20 @@ def get_all_configs() -> Mapping[str, Optional[Any]]:
 def get_raw_configs() -> Mapping[str, Optional[Any]]:
     try:
         all_configs = rds.hgetall(config_hash)
-        return {
+        configs = {
             k.decode("utf-8"): get_typed_value(v.decode("utf-8"))
             for k, v in all_configs.items()
             if v is not None
         }
+        if os.environ.get("SENTRY_SINGLE_TENANT"):
```

Comment:
Same as https://github.com/getsentry/sentry/pull/42097/files#r1041617089 do we want to use `SENTRY_ALLOW_LOCAL_CONFIG_OVERRIDE` maybe?

Reply:
https://github.com/getsentry/sentry/pull/42097#discussion_r1042537715